core/src/main/java/jenkins/management/CliLink.java

```diff
@@ -48,6 +52,12 @@ public String getDescription() {
         return Messages.CliLink_Description();
     }
 
+    @CheckForNull
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.MANAGE;
+    }
+
```

Comment:
No special permissions are needed to access this anyway.

Reply:
Maybe returning `Jenkins.READ` is a better option then. Could save time for JEP-224 later.
Same for the similar comments below

Reply:
Possibly, but the combined JEP-223/224 exists and solves this differently, since it needs explicit checks for either of the permissions in quite a few places.